src/main/scala/rocketchip/Periphery.scala

```diff
@@ -17,6 +17,11 @@ import rocket.XLen
 import scala.math.max
 import coreplex._
 
+case object MIFAddrBits extends Field[Int]
+case object MIFDataBits extends Field[Int]
+case object MIFTagBits extends Field[Int]
+case object MIFDataBeats extends Field[Int]
+
```

Comment:
I propose taking it a step further and actually eliminating all the MIF* parameter fields entirely. Keeping them around when the chip now supports multiple external interfaces seems like it can only lead to confusion down the road. They're basically just feeding the creation of the outermost NastiKey and nothing else. Ideally NastiKey, HastiKey and TLKey would all be handled in the same way when used to parameterize external interfaces.